scripts/benchmark_stats.py

```diff
@@ -31,16 +32,19 @@ def _process_file_data(self):
             with open(file) as fp:
                 for row in DictReader(fp, delimiter=","):
                     percentile_99th = int(row["99%"])
+                    request_count = int(row.get("Request Count") or row.get("# requests"))
+
                     if "/questionnaire" in row['Name']:
                         if row["Type"] == "GET":
-                            self.get_requests.append(percentile_99th)
+                            self.get_requests.append(percentile_99th * request_count)
+                            self.total_get_requests += request_count
                         elif row["Type"] == "POST":
-                            self.post_requests.append(percentile_99th)
+                            self.total_post_requests += request_count
+                            self.post_requests.append(percentile_99th * request_count)
 
                     if row["Name"] == "Aggregated":
-                        request_count = row.get("Request Count") or row.get("# requests")
                         failure_count = row.get("Failure Count") or row.get("# failures")
-                        self.total_requests += int(request_count)
+                        self.total_requests += request_count
```

Comment:
We should now remove this as we have the total get/post requests. This should also be removed from `__init__` and be made a property which is used by `average_total()` and `__str__()`. 
So something like:
```py
    @property
    def total_requests(self):
        return self.total_get_requests + self.total_post_requests
    
    @property
    def average_total(self):
        return sum(self.post_requests + self.get_requests)/self.total_requests
```
Might also be worth rounding to 2 decimals.

Reply:
Well you might think that, but how we have configured this means it removes the GET's for session and thank you, which I don't understand and why I didn't remove this. Kind of why I put a card in tech session as this repo needs some refining

Reply:
If we do round then consider just changing the display formatting rather than the value itself using the `{:0.2f}` formatting spec

Reply:
I will put rounding on the tech card, as well as testing